zeebe.js

```diff
@@ -0,0 +1,97 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
+async function assignUser([userTaskKey, assignee]) {
+    // Every request needs an access token.
+    const accessToken = await getAccessToken(authorizationConfiguration);
+  
+    // These settings come from your .env file.
+    const ZeebeApiUrl = process.env.ZEEBE_BASE_URL;
+    const userTaskKey = process.env.USER_TASK_KEY;
```

Comment:
```suggestion
```

Because we're now passing the `userTaskKey` as an argument to the function, we no longer need this line (and it causes an error due to defining `userTaskKey` twice).